frontend/packages/integration-tests-cypress/support/index.ts

```diff
@@ -20,6 +20,11 @@ Cypress.Cookies.defaults({
   preserve: ['openshift-session-token', 'csrf-token'],
 });
 
+Cypress.Commands.overwrite('log', (originalFn, message) => {
+  cy.task('log', `      ${message}`, { log: false }); // log:false means do not log task in runner GUI
```

Comment:
Don't we still want to see the message in the runner GUI?

Reply:
The next line `originalFn(message)` logs the original message.  the `log:false` avoid the logging of the `task` cmd.